wiki/Ranking_criteria/osu!catch/en.md

```diff
@@ -23,33 +24,30 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty. Rhythm
 
 #### Guidelines
 
-- **All circles and slider heads should represent a sound existing in the music.** This is usually a distinct sound, but can also represent a continuous sound with an indistinguishable start or end.
-- **Slider tails of extended sliders should be snapped according to the song's beat structure.** If the song is using a straight beat, 1/4, 1/8, and 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used. If the song has a sound in a different snap from what was recommended, snapping to an actual beat always takes priority.
+- **All [fruits](/wiki/Gameplay/Hit_object/Fruit) should represent a sound existing in the music.** This is usually a distinct sound but can also represent a continuous sound with an indistinguishable start or end.
+- **The last [fruit](/wiki/Gameplay/Hit_object/Fruit) of a [slider](/wiki/Gameplay/Hit_object/Juice_stream) should be [snapped](/wiki/Beatmapping/Snapping) according to the song's beat structure.** For example, if the song is using a straight beat, 1/4, 1/8 or 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used.
```

Comment:
these do have catch-specific terms

```suggestion
- **The last [fruit](/wiki/Gameplay/Hit_object/Fruit) of a [juice stream](/wiki/Gameplay/Hit_object/Juice_stream) should be [snapped](/wiki/Beatmapping/Snapping) according to the song's beat structure.** For example, if the song is using a straight beat, 1/4, 1/8 or 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used.
```

Reply:
maybe add parentheses for the canonical names i.e. `slider (a.k.a. juice stream)`